src/ptvsd/_vendored/pydevd/pydevd.py

```diff
@@ -1957,6 +1981,14 @@ def _wait_for_attach():
     py_db.block_until_configuration_done()
 
 
+def _is_attached():
+    '''
+    Can be called any time to check if the connection was established and the DAP (Debug Adapter Protocol) has sent
+    the ConfigurationDone request.
+    '''
+    py_db = get_global_debugger()
+    return bool(py_db) and py_db.is_attached()
```

Comment:
It doesn't make sense to use `bool()` in this context, since it's the same with or without it. If you want to make it a more narrow check, it should be `py_db is not None`.